src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -144,25 +157,38 @@ export class LargeArtistHeader extends Component<Props> {
                   </Serif>
                 </H2>
                 <Spacer mr={2} />
-                {props.artist.counts.follows > 50 && (
-                  <Serif size="3">
-                    {props.artist.counts.follows.toLocaleString()} followers
-                  </Serif>
-                )}
               </Flex>
             </Box>
-            <FollowArtistButton
-              useDeprecatedButtonStyle={false}
-              artist={props.artist}
-              user={user}
-              onOpenAuthModal={() =>
-                handleOpenAuth(props.mediator, props.artist)
-              }
-            >
-              Follow
-            </FollowArtistButton>
+            <Flex justifyContent="space-between">
+              {props.artist.counts.follows > 50 && (
+                <Flex flexDirection="column">
```

Comment:
In the spec it looks like these should be centered (but they don't appear that way in the screenshot).

Reply:
I accidentally uploaded the wrong screenshot for mobile my bad